.travis.yml

```diff
@@ -13,5 +13,6 @@ branches:
     - master
 
 script:
+  - npm run eslint
```

Comment:
We want Travis-CI to run eslint on pull requests so we can move the build and deploy commands to a separate script `bin/build_deploy`, then add the following to `.travis.yml`:

```yml
deploy:
  provider: script
  script: bin/build_deploy
  on:
    branch: master
```

Finally, enable PR building in Travis.